rally_ovs/plugins/ovs/scenarios/ovn_network.py

```diff
@@ -88,7 +88,16 @@ def create_and_bind_ports(self,
 
         sandboxes = self.context["sandboxes"]
 
-        lswitches = self._create_networks(network_create_args)
+        # For scale test, we need to also add more ports on demand by adding
+        # new HVs. Hence, we can skip creating lswitches and lrouters since
```

Comment:
the comment "lrouters" is misleading here. In fact the comment is not necessary, since the arg "use_existing_networks" is self-explaining.

Reply:
sure

Reply:
done